packages/@webex/plugin-meetings/src/multistream/remoteMediaManager.ts

```diff
@@ -49,6 +49,10 @@ export interface Configuration {
 
     layouts: {[key: LayoutId]: VideoLayout}; // a map of all available layouts, a layout can be set via setLayout() method
   };
+  receiveNamedMediaGroup?: {
```

Comment:
this is RemoteMediaManager class so it only deals with receiving of remote media, not sending, so the "receive" bit in the name is redundant. Maybe instead we should have "audioNamedMediaGroup", because it only applies to audio?

Reply:
It's ok to me to remove "receive".

Reply:
> Would it be more meanful in the Configuration definition with "receive" prefix, unless the config is only for recieving?
yes, the config is only for receiving, because the whole RemoteMediaManager class is only for remote media, so only for receiving

Reply:
BTW, why "value" is a number if it's supposed to describe the media type? this is very confusing, why isn't it named `mediaType` and isn't an enum?

Reply:
Sorry, should be the "type" member, which use the Homer side definition : ( 0 – RESERVED, 1 – Audio Language, 2 – Inter-meeting media exchange, 3 – Media origin, 4 – 255 – RESERVED)

Reply:
So I will remove the "receive" prefix